dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
+    return [] if level_source.nil?
+
+    responses = JSON.parse(level_source.data)
+    learning_module_weights = Hash.new(0)
+
+    responses.each do |level_id, response|
+      level = EvaluationMulti.find(level_id)
+      selected_answer = level.answers[response['result'].to_i]
```

Comment:
Should the `LevelGroup` own parsing the last attempt and exposing answers in a standardized way?  We now do this in two places: here and [in the `ApiController`](https://github.com/code-dot-org/code-dot-org/blob/0aaf55187a4b3439d403db68d589b2589e492eab/dashboard/app/controllers/api_controller.rb#L200).